lib/config/definitions.ts

```diff
@@ -102,6 +102,31 @@ export type RenovateOptions =
   | RenovateObjectOption;
 
 const options: RenovateOptions[] = [
+  {
+    name: 'allowedPostUpgradeTasks',
+    description:
+      'A list of regular expressions that determine which post-upgrade tasks are allowed. A task has to match at least one of the patterns to be allowed to run',
+    type: 'array',
+    subType: 'string',
+    default: [],
+    stage: 'global',
```

Comment:
```suggestion
    admin: true,
```

Should only be changeble by admin.